controller/script/destination-client/main.go

```diff
@@ -33,50 +34,71 @@ func main() {
 		Path:   *path,
 	}
 
-	rsp, err := client.Get(context.Background(), req)
-	if err != nil {
-		log.Fatal(err.Error())
-	}
-
-	for {
-		update, err := rsp.Recv()
-		if err == io.EOF {
-			break
-		}
+	if *method == "get" {
```

Comment:
From an organizational standpoint, my preference would be to use a switch statement here, and then move the existing get code into a separate function, and add a new function for the getProfile code. Something like:

```
	switch *method {
	case "get":
		get(client, req)
	case "getProfile":
		getProfile(client, req)
	default:
		log.Fatalf("Unknown method: %s; supported methods: get, getProfile", *method)
	}
```